static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -71,9 +71,11 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
           <Flex column style={{gridArea: 'right'}}>
             {''}
           </Flex>
-          <strong style={{gridArea: 'user'}}>
-            <FeedbackItemUsername feedbackItem={feedbackItem} detailDisplay={false} />
-          </strong>
+          <TextOverflow>
+            <strong style={{gridArea: 'user'}}>
+              <FeedbackItemUsername feedbackItem={feedbackItem} detailDisplay={false} />
+            </strong>
+          </TextOverflow>
```

Comment:
does it work without the extra `<strong>`? 

Looks like FeedbackItemUsername also includes `<strong>`, so we're really 🦾 🦾 

```suggestion
          <TextOverflow style={{gridArea: 'user'}}>
              <FeedbackItemUsername feedbackItem={feedbackItem} detailDisplay={false} />
          </TextOverflow>
```

Reply:
yup, i can remove the `<strong>` but I have to add a `<span>` in place of it to apply the style since the `TextOverflow` component doesn't accept that